examples/examples.py

```diff
@@ -95,18 +117,47 @@
 # Alternately, you can just use the Drum class that composes all these modules together automatically.
 
 # +
-my_drum = Drum()
+my_drum = Drum(
+    pitch_adsr=ADSR(0.25, 0.25, 0.25, 0.25, alpha=3),
+    amp_adsr=ADSR(0.25, 0.25, 0.25, 0.25),
+    vco=VCO(midi_f0=69, mod_depth=12),
+    sustain_duration=1
+    )
 
-boom = my_drum()
+drum_out = my_drum()
 
-X = librosa.stft(boom)
+X = librosa.stft(drum_out)
 Xdb = librosa.amplitude_to_db(abs(X))
 plt.figure(figsize=(5, 5))
 librosa.display.specshow(Xdb, sr=vca.sample_rate, x_axis="time", y_axis="hz")
 plt.ylim(0, 2000)
 plt.show()
 
-ipd.Audio(boom, rate=vca.sample_rate)
+ipd.Audio(drum_out, rate=vca.sample_rate)
 # -
+# You can also use the **SquareSaw oscillator** in the drum module.
+
+# +
+my_drum = Drum(
+    pitch_adsr=ADSR(0.25, 0.25, 0.25, 0.25, alpha=3),
+    amp_adsr=ADSR(0.25, 0.25, 0.25, 0.25),
+    vco=SquareSaw(shape=0, midi_f0=24, mod_depth=12),
+    sustain_duration=1
+    )
+
+drum_out = my_drum()
+
+X = librosa.stft(drum_out)
```

Comment:
How many times are we doing this? Maybe we should turn it into a function?

Reply:
meh.